src/main/java/fr/inria/diversify/mutant/pit/GradlePitTaskAndOptions.java

```diff
@@ -0,0 +1,58 @@
+package fr.inria.diversify.mutant.pit;
+
+/**
+ * Created by Daniele Gagliardi
+ * daniele.gagliardi@eng.it
+ * on 21/07/17.
+ */
+
+public class GradlePitTaskAndOptions {
+
+    public static boolean descartesMode = false;
+
+    public static boolean evosuiteMode = false;
+
```

Comment:
The two booleans fields are duplicates with The MavenPitCommandAndOptions. I think we can easily factorize the property and options between the two automatic builder using an abstract class, but it is not mandatory.